examples/aptos/call-contract/README.md

```diff
@@ -1,37 +1,38 @@
 # Call Contract
 
-Relay a message from source-chain to destination-chain.
+Relay a message bi-directional between Aptos and an EVM chain.
 
 Deploy:
 
 ```bash
-npm run deploy aptos/call-contract [local|testnet]
+npm run deploy aptos/call-contract local
```

Comment:
Causing error

```
$ npm run deploy aptos/call-contract local

> axelar-examples@1.0.0 deploy
> node scripts/runDeploy "aptos/call-contract" "local"

Deploying HelloWorld for Aptos.
/home/stephen/workspace/axelar/axelar-examples/node_modules/aptos/dist/index.js:5694
        throw new ApiError2(
              ^

ApiError2: {"message":"Account not found by Address(0x8ac1b8ff9583ac8e661c7f0ee462698c57bb7fc454f587e3fa25a57f9406acc0) and Ledger version(404)","error_code":"account_not_found","vm_error_code":null}
    at AptosNetwork.wrapper (/home/stephen/workspace/axelar/axelar-examples/node_modules/aptos/dist/index.js:5694:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Promise.all (index 0)
    at async AptosNetwork.generateRawTransaction (/home/stephen/workspace/axelar/axelar-examples/node_modules/aptos/dist/index.js:5478:91)
    at async AptosNetwork.generateSignSubmitTransaction (/home/stephen/workspace/axelar/axelar-examples/node_modules/aptos/dist/index.js:5500:28) {
  status: 404,
  errorCode: 'account_not_found',
  vmErrorCode: null
}
```

Reply:
@StephenFluin could you try adding this to `.env` file

```
APTOS_ADDRESS=0x8ac1b8ff9583ac8e661c7f0ee462698c57bb7fc454f587e3fa25a57f9406acc0
```

Reply:
What if they have an old copy of axelar-examples. Could we have the setup script fix this, or detect it and throw a better error?

Reply:
good idea. i've added a condition to check if all necessary variables are set in the `build.sh` file.
https://github.com/axelarnetwork/axelar-examples/pull/98/commits/b832653556c91df7b171859331674ca88b763365